api/src/test/java/org/openmrs/module/eptsreports/api/EptsReportsServiceTest.java

```diff
@@ -10,36 +10,18 @@
 package org.openmrs.module.eptsreports.api;
 
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
-import org.openmrs.module.eptsreports.api.dao.EptsReportsDao;
-import org.openmrs.module.eptsreports.api.impl.EptsReportsServiceImpl;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
 
 /**
  * This is a unit test, which verifies logic in EPTSreportsService. It doesn't extend
  * BaseModuleContextSensitiveTest, thus it is run without the in-memory DB and Spring context.
  */
-public class EptsReportsServiceTest {
-
-  @InjectMocks EptsReportsServiceImpl basicModuleService;
-
-  @Mock EptsReportsDao dao;
+public class EptsReportsServiceTest extends BaseModuleContextSensitiveTest {
 
   @Before
```

Comment:
sorry, this part should have been in MEPTS-175, but no worry, i can pick it up from here there.